mage_ai/data_preparation/templates/data_loaders/bigquery.py

```diff
@@ -19,3 +21,10 @@ def load_data_from_big_query(**kwargs) -> DataFrame:
     config_profile = 'default'
 
     return BigQuery.with_config(ConfigFileLoader(config_path, config_profile)).load(query)
+
+@test
+def test_load_data(df: DataFrame) -> None:
```

Comment:
We need a better way to ingest these common test template code. There're 20+ templates added in latest master branch. Since the test template code is all the same, we can use jinja template to render it.

Reply:
@wangxiaoyou1993 I updated the templates to use jinja templating. I'm not sure how much better it is than just manually copying the test code over, but it makes use of some jinja templating features